src/Features/CSharp/Portable/InitializeParameter/CSharpAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -92,5 +91,27 @@ protected override StatementSyntax CreateParameterCheckIfStatement(DocumentOptio
                 statement: ifTrueStatement,
                 @else: null);
         }
+
+        protected override async Task<Document?> TryAddNullCheckToParameterDeclarationAsync(Document document, IParameterSymbol parameter, CancellationToken cancellationToken)
+        {
+            var tree = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+            if (tree is null)
+            {
+                return null;
+            }
+
+            var options = (CSharpParseOptions)tree.Options;
+            if (options.LanguageVersion < LanguageVersion.Preview)
```

Comment:
We need to track this somehow. I think Julien has a apattern on how to reference a lang version without using 'preview'

Reply:
Sure. I was going to create a tracking issue to revisit usages of LangVersion.Preview before release. But maybe @jcouv's method would be better.

Reply:
I see that #54359 added `LanguageVersionFacts.CSharpNext`. But it seems it's not accessible from `Features`.

Reply:
You can add something similar to the IDE layer. If so, please add a comment next to CSharpNext so that we remember to fix both when adding a new language version.

Reply:
Done.